packages/config/config/devices/0x013c/pan06.json

```diff
@@ -143,6 +164,31 @@
 					"value": 2
 				}
 			]
+		},
+		{
+			"#": "11",
+			"label": "Send Reports From Endpoints",
+			"description": "Enable this to distinguish between the relays.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 1
+				},
+				{
+					"label": "Enable",
+					"value": 2
+				}
+			]
 		}
-	]
+	],
+	"compat": {
+		// Not required for >= 1.8
+		"preserveRootApplicationCCValueIDs": true
```

Comment:
I guess it doesn't hurt here, but should we find a better solution for this?
Or keep this file splitted?

Reply:
We have an open Issue to make labels and other things conditional. #3672